### PR TITLE
Front Matter of the MicroProfile Tutorial

### PR DESCRIPTION
**Pull request by ttelang**
- Incorporated changes to the Legal section as per previous PR

### CODE REVIEW

File: index.adoc [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+.MicroProfile e-Commerce Application
+image::images/figureFM-1.png[MicroProfile e-Commerce Application]
+
+As you can see in the above figure, together these microservices form a robust and flexible e-Commerce application architecture, enabling scalable, efficient, and secure online shopping experiences.
+
+=== Downloading the Code
+The code examples in this tutorial are available at this link:https://www.github.com/ttelang/microprofile-examples[repo] (TODO: to be changed).
+
+=== Prerequisites
+Microprofile use the Java Platform, and are usually written in the Java programming language.
```
Comment (Emily-Jiang) [inline index.adoc:81]:
```suggestion
MicroProfile use the Java Platform, and are usually written in the Java programming language. 
```

File: index.adoc [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+.. Gain a solid understanding of what MicroProfile is and its role in modern cloud-native application development.
+
+.. Learn about the evolution of MicroProfile and its relationship with Jakarta EE.
+
+.. Understand how MicroProfile facilitates building microservices.
+
+. Hands-On Experience with Key MicroProfile APIs:
+
+.. Learn to implement Config, Health Check, Metrics, JWT Authentication, Fault Tolerance, Rest Client, and other MicroProfile APIs.
```
Comment (Emily-Jiang) [inline index.adoc:102]:
```suggestion
.. Learn to implement Config, Health, Metrics, JWT Authentication, Fault Tolerance, Rest Client, and other MicroProfile APIs.
```

File: chapter01/chapter01.adoc [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+   * Emerson Castañeda
+
+Java user groups 
+
+   * Atlanta Java User Group (AJUG)
+   * Association of the German Java User Groups (iJUG)
+
+Corporate Members
+
+  * Hunan AsiaInfo Anhui
```
Comment (Emily-Jiang) [inline chapter01/chapter01.adoc:35]:
```suggestion
```

Comment (Emily-Jiang) [reply chapter01/chapter01.adoc:35]: It is no longer a working group member.

**Comment by ttelang** [comment]
I pushd some additional changes into this branch by mistake. Closing this as Created a new clearn PR https://github.com/ttelang/microprofile-tutorial/pull/9 containing only changes for Preface.

**Pull request closed without merging**